### DISCUSSION TITLE
No sound No default card (Cyrrus Logic WM5102, Rune 0.4b work well)

### DISCUSSION THREAD

**Discussion by vip5912** [rern/rAudio#62] [Bug Reports]
Card: Generic Cyrrus Logic WM5102

System:
rAudio 1 • 20210101
Raspberry Pi 2 Model B Rev 1.1
Generic Cirrus Logic WM5102

# cat /boot/config.txt

force_turbo=1
gpu_mem=32
initramfs initramfs-linux.img followkernel
max_usb_current=1
disable_splash=1
disable_overscan=1
dtparam=audio=off
dtparam=i2s=on
dtoverlay=rpi-cirrus-wm5102

# cat /etc/asound.conf

defaults.pcm.card 0
defaults.ctl.card 0

# aplay -l | grep ^card

card 0: RPiCirrus [RPi-Cirrus], device 0: WM5102 AiFi wm5102-aif1-0 [WM5102 AiFi wm5102-aif1-0]

# amixer scontrols

**Comment by rern** [comment]
Try today's update.
